Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -5,17 +5,21 @@ import XCTest
 final class MongoDatabaseTests: MongoSwiftTestCase {
     static var allTests: [(String, (MongoDatabaseTests) -> () throws -> Void)] {
         return [
-            ("testDatabase", testDatabase)
+            ("testDatabase", testMongoDatabase)
```

Comment:
I think we also need to update the string here.

Reply:
woops, fixed.